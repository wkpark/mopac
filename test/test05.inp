  ts  +
           am1
  Location of transition state for HCN -> HNC
  Check the heat of formation is 118.456 kcal/mol
  H
  C  1.3 1
  N  1.3 1 60 1

