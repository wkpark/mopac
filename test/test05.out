 *******************************************************************************
 ** FRANK J. SEILER RES. LAB., U.S. AIR FORCE ACADEMY, COLO. SPGS., CO. 80840 **
 *******************************************************************************

                                 AM1 CALCULATION RESULTS


 *******************************************************************************
 *          MOPAC:  VERSION  6.00               CALC'D. 31-OCT-90               
 *   T=      - A TIME OF  3600.0 SECONDS REQUESTED
 *  DUMP=N   - RESTART FILE WRITTEN EVERY  3600.0 SECONDS
 *  TS       - USE EF ROUTINE FOR TS SEARCH
 *  AM1      - THE AM1 HAMILTONIAN TO BE USED
 ***********************************************************************040BY040
  TS  +
           AM1
  Location of transition state for HCN -> HNC
  Check the heat of formation is 118.456 kcal/mol

    ATOM   CHEMICAL  BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL    (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                  NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      H       
      2      C         1.30000  *                                  1
      3      N         1.30000  *      60.00000  *                 2    1


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         H        0.0000    0.0000    0.0000
     2         C        1.3000    0.0000    0.0000
     3         N        0.6500    1.1258    0.0000
  H: (AM1): M.J.S. DEWAR ET AL, J. AM. CHEM. SOC. 107 3902-3909 (1985)          
  C: (AM1): M.J.S. DEWAR ET AL, J. AM. CHEM. SOC. 107 3902-3909 (1985)          
  N: (AM1): M.J.S. DEWAR ET AL, J. AM. CHEM. SOC. 107 3902-3909 (1985)          


          RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =  5


            INTERATOMIC DISTANCES
0
                  H  1       C  2       N  3
 ------------------------------------------
     H    1   0.000000
     C    2   1.300000   0.000000
     N    3   1.300000   1.300000   0.000000

          HESSIAN CALCULATED NUMERICALLY

 CYCLE:   1 TIME:   1.59 TIME LEFT:   3596.8 GRAD.:   173.109 HEAT: 124.5708    
 CYCLE:   2 TIME:   0.05 TIME LEFT:   3596.8 GRAD.:    54.731 HEAT: 118.9052    
 CYCLE:   3 TIME:   0.31 TIME LEFT:   3596.5 GRAD.:    11.440 HEAT: 118.4807    
 CYCLE:   4 TIME:   0.30 TIME LEFT:   3596.2 GRAD.:     0.402 HEAT: 118.4563    
 CYCLE:   5 TIME:   0.27 TIME LEFT:   3595.9 GRAD.:     0.280 HEAT: 118.4566    

 -------------------------------------------------------------------------------
  TS  +
           AM1
  Location of transition state for HCN -> HNC
  Check the heat of formation is 118.456 kcal/mol


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                                AM1    CALCULATION
                                                       VERSION  6.00
                                                       31-OCT-90               




          FINAL HEAT OF FORMATION =        118.45658 KCAL


          TOTAL ENERGY            =       -344.05301 EV
          ELECTRONIC ENERGY       =       -608.70637 EV
          CORE-CORE REPULSION     =        264.65336 EV

          IONIZATION POTENTIAL    =         11.60654
          NO. OF FILLED LEVELS    =          5
          MOLECULAR WEIGHT        =     27.026


          SCF CALCULATIONS  =                9
          COMPUTATION TIME =   2.970 SECONDS





    ATOM   CHEMICAL  BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL    (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                  NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      H       
      2      C         1.29745  *                                  1
      3      N         1.21638  *      67.61179  *                 2    1


            INTERATOMIC DISTANCES
0
                  H  1       C  2       N  3
 ------------------------------------------
     H    1   0.000000
     C    2   1.297451   0.000000
     N    3   1.400269   1.216381   0.000000


                  EIGENVALUES

 -37.30544 -18.51901 -15.85472 -13.47827 -11.60654   0.14744   1.18301   1.31156
   5.10289


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

         ATOM NO.   TYPE          CHARGE        ATOM  ELECTRON DENSITY
           1          H           0.2971          0.7029
           2          C          -0.0916          4.0916
           3          N          -0.2055          5.2055
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -1.394    -1.110     0.000     1.782
 HYBRID        -0.635     0.308     0.000     0.706
 SUM           -2.029    -0.802     0.000     2.182


          CARTESIAN COORDINATES 

    NO.       ATOM               X         Y         Z

     1         H                  0.0000    0.0000    0.0000
     2         C                  1.2975    0.0000    0.0000
     3         N                  0.8342    1.1247    0.0000


          ATOMIC ORBITAL ELECTRON POPULATIONS

   0.70292   1.71165   0.66809   0.99358   0.71829   1.83592   0.90276   1.18508
   1.28171
  CHANNEL 12 OPENED



 TOTAL CPU TIME:             3.23 SECONDS

 == MOPAC DONE ==
